README.md

```diff
@@ -83,7 +83,8 @@
 
 <!-- > Add a link to your deployed project. -->
 
-- [Live Demo Link]()
+- [Link1](weather-dashboard-delta.vercel.app)
+- [Link2](https://develop--lambent-beignet-32b81d.netlify.app)
 
```

Comment:
- [ ] [OPTIONAL] Kindly consider adding the video link to the README file so it can be accessible to anyone with the with the access to your repo